lib/policy_machine_storage_adapters/template.rb

```diff
@@ -173,5 +173,11 @@ def scoped_privileges(user_or_attribute, object_or_attribute)
 
     end
 
+    # Optimized version of PolicyMachine#accessible_objects
+    # Return all objects the user has the given operation on
+    # Optional: only add this method if you can do it better than policy_machine.rb
+    def accessible_objects(user_or_attribute, operation, options = {})
```

Comment:
I think it might've been Matt's idea--I want to blame it on him but the git history of this file doesn't work in my favor. I guess the point was that copying a file and filling in method definitions was easier with a template than with a module, and superclasses would prevent inheriting from like a Datamapper class or something. I've never done it this way before or since though.